src/gt4sd/domains/materials/scorer.py

```diff
@@ -216,6 +216,17 @@ def score(self, smiles: str) -> float:
         )
         return scoring_function.score_mol(Chem.MolFromSmiles(smiles))
 
+    def score_list(self, smiles_list: List[str]) -> List[float]:
```

Comment:
why not defining this in `TargetValueScorer` so we avoid repeating the same in every child class.

Reply:
Just noticed it was already defined, I was not using it directly from parent, I will remove these methods from the child classes, thanks :)